### PR TITLE
Feat/better result return in excute tx

### PR DESCRIPTION
**Pull request by vuvth**
## Proposed Changes

Return list receipt when run algob.exeucteTx

## Check list: 
- [ ] Should I update web.executeTx too ?
- [ ] Test is good 
- [x] Update documents

### CODE REVIEW

File: packages/algob/test/fixture-projects/stateful/artifacts/.cp.yaml [hunk 1]
```diff
@@ -1,25 +1,17 @@
 network1:
-  timestamp: 1623663397441
```
Comment (vuvth) [inline packages/algob/test/fixture-projects/stateful/artifacts/.cp.yaml:2]: Why this file change?

**Review by sczembor — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,6 +48,14 @@ Added:
 
 - Added support for saving smart contract template params in ASCCache.
 
+- Return list of receipt for each txn in group txn. Example:
```
Comment (sczembor) [inline CHANGELOG.md:51]:
```suggestion
- Return list of receipts for each txn in group txn. Example:
```

**Review by sczembor — APPROVED** [review]
LGTM

**Pull request merged**